### DISCUSSION TITLE
connect through SAPROUTER

### DISCUSSION THREAD

**Discussion by juanele121976** [SAP/node-rfc#305] [Q&A] [answered]
Hello, I would like to ask how I connect through SAP Router.. how to configure the 
sapnwrfc.ini..??
first of all, Thanks

**Comment by bsrdjan** [comment]
Connection parameters for SAP router, load balancing etc are documented in `sapnwrfc.ini` file, located in SAP NW RFC SDK `demo` folder. Just search for SAPROUTER, there should be couple of examples there.

**Comment by juanele121976** [comment, marked as answer]
Hello, thank you very much for your support. I reviewed it and was able to connect
Greetings,
